Resources/Prototypes/Nyanotrasen/Entities/Clothing/Neck/mantles.yml

```diff
@@ -0,0 +1,4 @@
+- type: entity
+  parent: ClothingNeckMantleRD
+  id: ClothingNeckMantleMystagogue
+  name: mystagogue's mantle
```

Comment:
There's an issue with this item. Its parent is already mapped into a few of our maps. There needs to be a decision made whether to continue using it and renaming the upstream item or to make a new one entirely. The decision should apply in general. For example, I renamed the upstream item when making the mystagogue's signet ring, but we don't have to continue that trend. It is probably best to move all these items out to their own files if they're going to be Nyano-specific derivatives.

This might be one of those times we can get some mileage out of migrations.